### ISSUE TITLE
3.1.0-beta4 - Startup tab of function node is too tall

### ISSUE THREAD

**Issue by TotallyInformation** [node-red/node-red#4296]
### Current Behavior

![image](https://github.com/node-red/node-red/assets/1591850/8613e4a4-3689-4cab-b179-bb48505a6a6f)

Those with sharp eyes will note that the bottom of the tab display is hidden and there is an unexpected scroll bar. This is the current version of Edge on W11. The browser window is 100% zoom and the Editor zoom is set to default.

Oddly, the tray has a fixed height which I'm guessing is the problem. It would be much better with at least a height based on rem or em instead of px. (This is actually true of a lot of the editor and it would be nice to see that fixed in future releases).

![image](https://github.com/node-red/node-red/assets/1591850/876b28a5-2fe1-45bb-81ad-262b1eba5437)

[Steve-Mcl](https://discourse.nodered.org/u/Steve-Mcl)
[41m](https://discourse.nodered.org/t/node-red-3-1-0-beta-4-released/80091/95?u=totallyinformation)
I suspect the add button has been pushed off screen with the addition of the timeout. If I remember correctly the height of the form elements are totted up and the setup imports div is sized to the remaining space. The onresize handler probably needs updating to accommodate the timeout row?

In short, it's a bug and it would be really good if you could capture it in an issue please.

### Expected Behavior

There should not be any scroll bar

### Steps To Reproduce

Open the setup tab in any function node

### Environment

- Node-RED version: 3.1.0-beta4
- Node.js version: v18
- npm version: N/A
- Platform/OS: Win 11
- Browser: Edge
